SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -137,9 +136,6 @@ CSU2TCLib::CSU2TCLib(const CConfig* config, unsigned short val_nDim, bool viscou
     Tcb_b.resize(nReactions,0.0);
 
     /*--- Assign gas properties ---*/
-    // Wall mass fractions for catalytic boundaries
-    Wall_Catalycity[0] = 0.999;
```

Comment:
So did we make a decision on where to keep this data? I think in my very stale for the cat wall branch I had it as a config option, which I think is a good choice, but would it be better to store it here in fluid model? I think I remember getting push back for having it as a config option.

Reply:
Hmmm, I do recall the issue.  I'm not sure what's the best way forward here.  Either way, hardcoding values doesn't seem like the right move.  (If I understand catalyticity correct....)

Reply:
Chatted about this, I think we agreed to use the free-stream values as the default, rather than having hard-coded values in fluid model, so I agree with removal here.